.vscode/settings.json

```
@@ -0,0 +1,12 @@
+// Document用設定ファイル
+{
+    // d2 のレイアウトエンジンの指定
+    "D2.previewLayout": "elk",
+    "[d2]": {
+        // d2ファイルのタブスペース数の指定
+        "editor.tabSize": 4
```

タブ文字混入を避けるために`editor.insertSpaces`も設定しておきませんか？

```suggestion
        "editor.tabSize": 4,
        "editor.insertSpaces": true
```